torch/utils/_config_module.py

```diff
@@ -310,6 +325,15 @@ def __delattr__(self, name: str) -> None:
         self._config[name].user_override = _UNSET_SENTINEL
         self._config[name].hide = True
 
+    def _get_alias_val(self, name: str) -> Optional[Any]:
+        alias = self._config[name].alias
+        if alias is None:
+            return None
+        module_name, constant_name = alias.rsplit(".", 1)
+        module = importlib.import_module(module_name)
```

Comment:
Is it worth catching the import exception to give it a slightly better message?

Reply:
what would that look like?

Reply:
Something like
```
try:
import_module
Except e:
  raise AttributeError("config alias {alias} does not exist) from e
```